app/models/manageiq/providers/ibm_terraform/configuration_manager.rb

```diff
@@ -64,12 +68,12 @@ def edit_with_params(params, endpoints, authentications)
         ems.endpoints = endpoints.map(&method(:assign_nested_endpoint))
         ems.authentications = authentications.map(&method(:assign_nested_authentication))
 
+        ems.ensure_provider_zone_id
```

Comment:
Typically we'll call this with a `before_update` callback on the class like `before_update :ensure_provider_zone_id`